bin/build.sh

```diff
@@ -0,0 +1,10 @@
+for example_file in exercises/**/*.example
+do
+  exercise_dir=$(dirname $example_file)
+  exercise=$(basename $example_file .example)
+  mv "$exercise_dir/$exercise.elm" "$exercise_dir/$exercise.impl"
+  mv "$exercise_dir/$exercise.example" "$exercise_dir/$exercise.elm"
+  elm-test exercises/**/*Tests.elm
+  mv "$exercise_dir/$exercise.elm" "$exercise_dir/$exercise.example"
+  mv "$exercise_dir/$exercise.impl" "$exercise_dir/$exercise.elm"
```

Comment:
I love love love the use of `.example` as the file extension, and rename to `.elm`. That's really clean!